collections/_posts/2022-06-13-CompactConvolutionalTransformer.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: review
+title: "Escaping the big data paradigm with compact transformers"
+tags: CNN transformer classification
+author: "Olivier Bernard"
+cite:
+    authors: "Ali Hassani, Steven Watson, Nikhil Shah, Abulikemu Abuduweili, Jiachen Li, Humphrey Shi"
+    title:   "Escaping the big data paradigm with compact transformers"
+    venue:   "arXiv"
+pdf: "https://arxiv.org/abs/2104.05704"
+---
+
+# Notes
+
+* This paper was accepted in a workshop at CVPR 2021
```

Comment:
I also suggested to delete the line I commented on since it's now redundant with the information provided in the header. Sorry for the nitpick :stuck_out_tongue:

Reply:
Done :)